canonical/ubuntu.com
# Upgrade to Vanilla 4.3

## Done

- Updates Vanilla to 4.3
- Updates cookie policy to 3.5 (to support Vanilla 4)
- Updates width of global nav (to fit new grid width in Vanilla 4)
- **DOES NOT** update global nav version (as this is tied to meganav work)

Drive-by: 
- added missing root element for logo section on the home page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- ⚠️ Review as many pages/templates as possible against [Vanilla migration guide](https://vanillaframework.io/docs/migration-guide-to-v4)
- Review the pages with broken logo strips and make sure they look decent (no need to be perfectly aligned, but just not look broken):
      - **NOTE**: logo sections are meant to be left aligned now, so the fact of them not being centered is not a bug
      - https://ubuntu-com-13131.demos.haus/embedding
      - https://ubuntu-com-13131.demos.haus/financial-services
      - https://ubuntu-com-13131.demos.haus/internet-of-things
      - https://ubuntu-com-13131.demos.haus/internet-of-things/appstore
      - https://ubuntu-com-13131.demos.haus/kubernetes
      - https://ubuntu-com-13131.demos.haus/openstack/support
      - https://ubuntu-com-13131.demos.haus/automotive
      - https://ubuntu-com-13131.demos.haus/azure
      - https://ubuntu-com-13131.demos.haus/robotics/ros-esm
      - https://ubuntu-com-13131.demos.haus/robotics/what-is-ros
      - https://ubuntu-com-13131.demos.haus/internet-of-things/smart-city
      - https://ubuntu-com-13131.demos.haus/telco/vnf
      - https://ubuntu-com-13131.demos.haus/telco/cnf
      - https://ubuntu-com-13131.demos.haus/telco/openran
      - https://ubuntu-com-13131.demos.haus/telco/magma
      - https://ubuntu-com-13131.demos.haus/kubernetes/what-is-kubernetes
      - https://ubuntu-com-13131.demos.haus/landscape
      - https://ubuntu-com-13131.demos.haus/robotics
      - https://ubuntu-com-13131.demos.haus/security
      - https://ubuntu-com-13131.demos.haus/ai
      - https://ubuntu-com-13131.demos.haus/ai/what-is-kubeflow
      - https://ubuntu-com-13131.demos.haus/ceph/what-is-ceph
      - https://ubuntu-com-13131.demos.haus/managed
      - https://ubuntu-com-13131.demos.haus/openstack
      - https://ubuntu-com-13131.demos.haus/openstack/managed
      - https://ubuntu-com-13131.demos.haus/core
      - https://ubuntu-com-13131.demos.haus/desktop/features
      - https://ubuntu-com-13131.demos.haus/telco
      - https://ubuntu-com-13131.demos.haus/openstack/consulting
      - https://ubuntu-com-13131.demos.haus/openstack/features
      - https://ubuntu-com-13131.demos.haus/aws
      - https://ubuntu-com-13131.demos.haus/edge-computing
      - https://ubuntu-com-13131.demos.haus/gov
      - https://ubuntu-com-13131.demos.haus/hpc
      - https://ubuntu-com-13131.demos.haus/hpc/use-cases
      - https://ubuntu-com-13131.demos.haus/managed/apps/kafka
      - https://ubuntu-com-13131.demos.haus/managed/apps
      - https://ubuntu-com-13131.demos.haus/observability
      - https://ubuntu-com-13131.demos.haus/observability/managed
      - https://ubuntu-com-13131.demos.haus/security/livepatch
      - https://ubuntu-com-13131.demos.haus/server
      - https://ubuntu-com-13131.demos.haus/rfp

## Issue / Card

Replaces previous testing PR #12791



## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
